app/helpers/application_helper.rb

```diff
@@ -1,4 +1,9 @@
 # frozen_string_literal: true
 
 module ApplicationHelper
+  def show_svg(path)
```

Comment:
If you don't have a workaround, you can just disable rubocop here
```suggestion
  # rubocop:disable Rails/OutputSafety
  def show_svg(path)
```